exercises/e-data02-derive-master.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>Upon review, we think that this task is not terribly useful or realistic; why not just use <code>tf_count()</code>? Could use a simpler / more realistic example, e.g. the <code>df_grades</code> example I'm sending separately.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdelrosario/evc-course/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>